source/free-monitoring.txt

```diff
@@ -0,0 +1,33 @@
+.. _free-monitoring-mongosh:
+
+===============
+Free Monitoring
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 1.4.1
+
+``mongosh`` prompts you to enable MongoDB's free monitoring service when 
+you connect to MongoDB Community Edition cluster.
```

Comment:
[suggestion] I know we are largely deferring to the server docs for more information, but it would still be helpful to explain at a high level what we are monitoring, so users who arrive at this page can see at glance whether this is something they're interested in.

Consider adding a sentence like:

The monitoring service provides information on your deployment such as operation execution times and memory usage.

Reply:
Thanks @jeff-allen-mongo I have added your suggestion.